test/core/src/temalab/communicator/Communicator.java

```diff
@@ -83,7 +83,7 @@ public void registerUnit() {
 	}
 
 
-	public void communicate() {
+	public void communicate() throws RuntimeException {
```

Comment:
`RuntimeException`-t nem kell kiírni, meg amúgy is lehetne helyette valami egyértelműbb.